.github/workflows/deploy-dev-testnet.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy Widgets to Testnet Dev
+on:
+  push:
+    branches: [develop]
```

Comment:
How is develop branch suppose to work? When we are merging to develop and when to main?

Reply:
This takes inspiration from near-discovery-components (see the full exploration [here](https://github.com/near/neardevhub-widgets/issues/304) ). They have a weekly promotion from develop branch to main automated on every Thursday. We can add this too, but I thought to leave that until after this was tested and working.

The general idea is that there is a testnet deploy of the develop branch, so we can run testing of new features on it.
Then there is a testnet deploy of the main branch, so we can investigate issues.
Then a mainnet deploy of main branch, for production.

Reply:
I see, a weekly staging for release would be great! Thank you!